src/edu/ucsb/nceas/metacat/properties/BackupPropertiesDelegate.java

```diff
@@ -0,0 +1,284 @@
+        if (bypassAlreadyChecked) {
+            logMetacat.debug(
+                "canBypass() returning false, since already previously checked");
+        } else {
+            // check how dev.runConfiguration is set in metacat.properties
+            String strRunConfiguration = PropertyService.getProperty("dev.runConfiguration");
+            logMetacat.debug(
+                "canBypass(): 'dev.runConfiguration property' set to: " + strRunConfiguration);
+            boolean runConfiguration = Boolean.parseBoolean(strRunConfiguration);
+
+            // MB: seems like this should be `bypassAlreadyChecked = true`, now we've checked it,
+            // but this was the original logic, so leaving as-is for now...
+            bypassAlreadyChecked = runConfiguration;
+
+            result = !runConfiguration;
+        }
+        return result;
+    }
+
+    /**
+     * Bypasses the metacat properties configuration utility. (Dev use only)
+     */
+    protected void bypassConfiguration() {
+        try {
+            if (!canBypass()) {
+                logAndThrow(
+                    "Attempting to do bypass when system is not configured for it.", null);
+            }
+            // The system is bypassing the configuration utility. We need to
+            // get the backup properties and replace existing properties with
+            // backup values.  We do this for main and org properties.
+            logMetacat.debug(
+                "bypassConfiguration: setting main backup properties.");
+            SortedProperties mainBackupProperties = getMainBackupProperties();
+            Vector<String> backupPropertyNames = mainBackupProperties.getPropertyNames();
+            for (String backupPropertyName : backupPropertyNames) {
+                String value = mainBackupProperties.getProperty(backupPropertyName);
+                PropertyService.setPropertyNoPersist(backupPropertyName, value);
+            }
+            logMetacat.debug(
+                "bypassConfiguration: setting configutil sections to true.");
+            PropertyService.setPropertyNoPersist("configutil.propertiesConfigured", "true");
+            PropertyService.setPropertyNoPersist("configutil.authConfigured", "true");
+            PropertyService.setPropertyNoPersist("configutil.skinsConfigured", "true");
+            PropertyService.setPropertyNoPersist("configutil.databaseConfigured", "true");
+            PropertyService.setPropertyNoPersist("configutil.geoserverConfigured", "bypassed");
+
+            PropertyService.persistProperties();
+
+        } catch (PropertyNotFoundException pnfe) {
+            logMetacat.error(
+                "bypassConfiguration: Could not find property: " + pnfe.getMessage());
+        } catch (GeneralPropertyException gpe) {
+            logMetacat.error(
+                "bypassConfiguration: General property error: " + gpe.getMessage());
+        }
```

Comment:
please explain why we can absorb these exceptions

Reply:
PropertyService, AuthPropertiesDelegate & BackupPropertiesDelegate addressed in commit 7a41ed9274d9c0e63125b03add225c38537dae91